modules/nativoBidAdapter.js

```diff
@@ -145,7 +145,7 @@ export const spec = {
         'params.url',
       )
       if (pageUrl == undefined || pageUrl === '') {
-        pageUrl = bidderRequest.refererInfo.page
+        pageUrl = bidderRequest.refererInfo.location
```

Comment:
what's wrong with page? We spent a lot of time on the logic of it

Reply:
Hi @patmmccann, we need the full URL including any query params which is why we opted for **location** over **page** so we would retain the entire url value